test/flows/uk_benefits_abroad_flow_test.rb

```diff
@@ -0,0 +1,1328 @@
+          add_responses going_or_already_abroad?: "already_abroad",
+                        which_country?: "ireland"
+          assert_next_node :is_british_or_irish?, for_response: "yes"
+        end
+
+        should "have a next node of db_already_abroad_gibraltar_outcome for a 'yes' response if country is gibraltar and already_abroad" do
+          add_responses going_or_already_abroad?: "already_abroad",
+                        which_country?: "gibraltar"
+          assert_next_node :db_already_abroad_gibraltar_outcome, for_response: "yes"
+        end
+
+        should "have a next node of worked_in_eea_or_switzerland? for a 'yes' response if country is in EEA and is not ireland or gibraltar and already_abroad" do
+          add_responses going_or_already_abroad?: "already_abroad"
+          assert_next_node :worked_in_eea_or_switzerland?, for_response: "yes"
+        end
+
+        should "have a next node of db_already_abroad_other_outcome for a 'no' response if already_abroad" do
+          add_responses going_or_already_abroad?: "already_abroad"
+          assert_next_node :db_already_abroad_other_outcome, for_response: "no"
+        end
+      end
+    end
+
+    context "question: is_claiming_benefits?" do
+      setup do
+        testing_node :is_claiming_benefits?
+        add_responses going_or_already_abroad?: "going_abroad",
+                      which_benefit?: "income_support",
+                      db_how_long_abroad?: "permanent",
+                      is_how_long_abroad?: "is_under_a_year_other"
+      end
+
+      should "render the question" do
+        assert_rendered_question
+      end
+
+      context "next_node" do
+        should "have a next node of is_claiming_benefits_outcome for any response" do
+          assert_next_node :is_claiming_benefits_outcome, for_response: "pension_premium"
+        end
+
+        should "have a next node of is_either_of_the_following? for an empty response" do
+          assert_next_node :is_either_of_the_following?, for_response: ""
+        end
+      end
+    end
+
+    context "question: is_either_of_the_following?" do
+      setup do
+        testing_node :is_either_of_the_following?
+        add_responses going_or_already_abroad?: "going_abroad",
+                      which_benefit?: "income_support",
+                      db_how_long_abroad?: "permanent",
+                      is_how_long_abroad?: "is_under_a_year_other",
+                      is_claiming_benefits?: ""
+      end
+
+      should "render the question" do
+        assert_rendered_question
+      end
+
+      context "next_node" do
+        should "have a next node of is_abroad_for_treatment? for any response" do
+          assert_next_node :is_abroad_for_treatment?, for_response: "too_ill_to_work"
+        end
+
+        should "have a next node of is_any_of_the_following_apply? for an empty response" do
+          assert_next_node :is_any_of_the_following_apply?, for_response: ""
```

Comment:
minor aside; 😬  the grammar of that node name - unless `is` is an abbreviation

Reply:
😬